basics/uattributes.adoc

```diff
@@ -102,6 +102,7 @@ The following table defines attributes that are used for publish messages in add
 |yes
 |A link:uri.adoc[UUri] representing the topic that this message is published to.
 
+* The UUri's `resource_id` *MUST* be set to a value in range `[0x8000, 0xFFFF[`.
```

Comment:
```suggestion
* The UUri's `resource_id` *MUST* be set to a value in range `[0x8000, 0xFFFF]`.
```

Reply:
My bad, this is supposed to be the (open) interval `[0x8000, 0xFFFF)`, using mathematical notation ...